rotkehlchen/chain/bitcoin/xpub.py

```diff
@@ -91,13 +89,11 @@ def _derive_addresses_loop(
             have_tx_mapping = have_bch_transactions([x[1] for x in batch_addresses])
         should_continue = False
         for idx, address in batch_addresses:
-            have_tx, balance = have_tx_mapping[address]
-            if have_tx:
+            if have_tx_mapping[address]:
```

Comment:
undo this all. This can't be removed.

Do you understand what it is you are removing?

This entire logic you are removing here is the derivation of addresses from xpubs. This needs to happen.Plus we have a background task that runs on them and calculates them